modules/repository/init.go

```diff
@@ -316,14 +316,13 @@ func initRepoCommit(ctx context.Context, tmpPath string, repo *repo_model.Reposi
 		return fmt.Errorf("git add --all: %w", err)
 	}
 
-	cmd := git.NewCommand(ctx,
-		"commit", git.CmdArg(fmt.Sprintf("--author='%s <%s>'", sig.Name, sig.Email)),
-		"-m", "Initial commit",
-	)
+	cmd := git.NewCommand(ctx, "commit").
+		AddOptionFormat("--author='%s <%s>'", sig.Name, sig.Email).
+		AddOptionValues("-m", "Initial commit")
```

Comment:
```suggestion
	cmd := git.NewCommand(ctx, "commit", "-m", "Initial commit").
		AddOptionFormat("--author='%s <%s>'", sig.Name, sig.Email)
```

Reply:
It just came from old code. I won't touch old logic too much. To make sure the code is clear enough during reviewing.